Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -701,29 +701,38 @@ set_option linter.deprecated false in
 theorem lift.initialSeg_coe : (lift.initialSeg.{u, v} : Ordinal → Ordinal) = lift.{v, u} :=
   rfl
 
-/-! ### The first infinite ordinal `omega` -/
+/-! ### The first infinite ordinal ω -/
 
 
 /-- `ω` is the first infinite ordinal, defined as the order type of `ℕ`. -/
```

Comment:
The docstring should explain why this is called `omega0` (to match `aleph0` and to make room for the soon to be added `Ordinal.omega` function)